### ISSUE TITLE
I have a problem with this

### ISSUE THREAD

**Issue by CannonPanda** [stnkl/EverythingToolbar#233]
I had installed the toolbar by following the order and I have win10&4.8 frame, but still got some problem, can you help me to solve this, than you! 
I have record my question here:  https://youtu.be/s7oTNBra7LM

**Comment by stnkl** [comment]
Please post the log file located at `%TEMP%\EverythingToolbar.log` and show a screenshot of the version number in Everything.

**Comment by CannonPanda** [comment]
![8b2ce86c6b12ad5ab12e6587a1b2ab7](https://user-images.githubusercontent.com/31563877/164973256-4ab73551-31d8-4d9f-b6cf-59087bad327f.png)
thanks

**Comment by stnkl** [comment]
Really interesting problem. The fact that EverythingToolbar shows 32 search results in the top right indicates that it actually gets the search results correctly but just can't display them for some reason. I am not sure why this would happen but maybe it helps if you would try two things for me:

1. Please check in the EverythingToolbar settings (three dot menu) if any item in the submenu "View" is checked and if it helps if you select a different item in there. This should change the level of detail of the search result items.
![grafik](https://user-images.githubusercontent.com/17520641/165151218-cab76f16-ec4a-4c9b-a6e1-2d6d5ecf64d8.png)

2. Make sure "Hide empty search" is unchecked in the settings and remove all content from the search box (remove whatever you typed to open EverythingToolbar) and see if that gives you any search results.

Depending on what happens here this _might_ give us an indication of where to look next.

Thank you!

**Comment by CannonPanda** [comment]
Hi Stephan, I got a good news for you. I have fixed the problem. 
The reason I met this is because I downloaded and installed ver 0.2.3 from a webpage a few days ago, but I forgot to run everthing at the moment. Then I unistalled it and download ver 0.7.4, and installed it with everthing running. Of course it didn't work, then I uninstalled and installed ver 0.7.4 several times, failed. 
Today I realized maybe it's related to the old version. So I uninstalled 0.7.4 and restarted my computer, then installed 0.2.3 with everthing running, it works, my thought was right. After this, I unistalled 0.2.3, restarted computer again, installed 0.7.4, succeeded. 
Thanks for your help and this helpful tool.
